### PR TITLE
Fix server 2012 use

### PR DESCRIPTION
**Pull request by roncain**
I've found that our certificates behaved differently on Server 2012, which I believe is why the CI machine failed the OuterLoop tests.  These tests now pass on a similarly configured Server 2012 VM.

### CODE REVIEW

**Comment by roncain** [comment]
Buddy-builders -- be aware an OuterLoop run might fail the first time, due to http.sys having port 44285 associated with a cert hash that no longer exists.  If so, it **should** pass the 2nd time.  If not, you can run "netsh http delete sslcert ipport=0.0.0.0:44285" to cleanup that issue.  The new code unconditionally removes that port from the SSL cert list when the Bridge completes, so it should "self repair" the stale state.

**Comment by iamjasonp** [comment]
Buddy built and passes. 
LGTM

**Comment by roncain** [comment]
Merging with intent to run CI OuterLoops

File: src/System.Private.ServiceModel/tools/test/SelfHostWcfService/Certificates/README.txt [hunk 1]
```diff
@@ -2,16 +2,23 @@
 =======================
```
Comment (dmetzgar) [inline src/System.Private.ServiceModel/tools/test/SelfHostWcfService/Certificates/README.txt]: Just curious why this readme file is a .txt and not a .md

File: src/System.Private.ServiceModel/tools/test/SelfHostWcfService/Certificates/README.txt [hunk 2]
```diff
@@ -20,7 +27,20 @@ The self-signed certificate for SSL stored in the "My" store
...
+      makecert -sk "RootCAWcfBridge" -iv RootCAWcfBridge.pvk -n "CN=localhost" -ic RootCAWcfBridge.cer -sr localmachine -ss my -sky exchange -pe
+    Then start the certificate manager and do these steps:
+      1. Under the "Personal"folder find the "localhost" certificate whose issuer is the "DO_NOT_TRUST_RootCAWcfBridge" above
+      2. Right-click | All Tasks | Export
+      3. Check "Export private keys" | NEXT
+      4. NEXT
+      5. When asked for a name to save, choose RootCAWcfBridge.pfx in the same folder as RootCAWcfBridge.cer
+
+The reason for this use of the certificate manager UI is that makecert writes the .pfx file as UNICODE,
+but certificate manager writews it as ASCII. If written as UNICODE it fails to load properly.
```
Comment (dmetzgar) [inline src/System.Private.ServiceModel/tools/test/SelfHostWcfService/Certificates/README.txt]: NIT: writews
Also put a space between "Personal" and folder

**Pull request merged**